Source/Core/DolphinWX/NetWindow.cpp

```diff
@@ -278,6 +278,31 @@ void NetPlaySetupDiag::OnHost(wxCommandEvent&)
 	}
 }
 
+void NetPlaySetupDiag::HostGame(std::string game, unsigned long port) {
```

Comment:
Brace on next line, also `game` can be a `const std::string&` assuming nothing modifies it in the function.